packages/website-eslint/src/linter/parser.js

```diff
@@ -0,0 +1,46 @@
+import { analyze } from '@typescript-eslint/scope-manager/dist/analyze';
+import { visitorKeys } from '@typescript-eslint/visitor-keys/dist/visitor-keys';
+import { astConverter } from '@typescript-eslint/typescript-estree/dist/ast-converter';
+import { createASTProgram } from './create-ast-program.js';
+import { extra } from './config.js';
+
+function parseAndGenerateServices(code, options) {
+  const { ast, program } = createASTProgram(code, options);
+  const { estree, astMaps } = astConverter(
+    ast,
+    { ...extra, code, jsx: options.jsx ?? false },
+    true,
+  );
+  return {
+    ast: estree,
+    services: {
+      hasFullTypeInformation: true,
+      program,
+      esTreeNodeToTSNodeMap: astMaps.esTreeNodeToTSNodeMap,
+      tsNodeToESTreeNodeMap: astMaps.tsNodeToESTreeNodeMap,
+    },
+  };
+}
+
+export function parseForESLint(code, parserOptions) {
+  const { ast, services } = parseAndGenerateServices(code, {
+    ...parserOptions,
+    jsx: parserOptions.ecmaFeatures?.jsx ?? false,
+    useJSXTextNode: true,
+    projectFolderIgnoreList: [],
+  });
+  const scopeManager = analyze(ast, {
+    ecmaVersion:
+      parserOptions.ecmaVersion === 'latest' ? 1e8 : parserOptions.ecmaVersion,
+    globalReturn: parserOptions.ecmaFeatures?.globalReturn ?? false,
+    sourceType: parserOptions.sourceType ?? 'script',
+  }); // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
```

Comment:
extraneous comment?
there's no code on the next line that gets ignored by this

Reply:
i missed that when i extracted build from website, good catch